src/lib/select/select.md

```diff
@@ -1,8 +1,11 @@
+Angular material supports both native html select and `<mat-select>` to work inside a 
```

Comment:
I would leave the original text for this file the same, and then add a new paragraph underneath that says something like
```markdown
Angular Material also supports use of native the `<select>` element inside of
`<mat-form-field>`. The native control has several performance, accessibility,
and usability advantages. See [the documentation for
form-field](https://material.angular.io/components/form-field) for more information.
```